duneana/AnaTree/AnalysisTree.fcl

```diff
@@ -11,6 +11,7 @@ dune35t_analysistree:
  CryGenModuleLabel:           "generator"
  ProtoGenModuleLabel:         ""
  G4ModuleLabel:               "largeant"
+ SimChannelLabel:             "largeant" # For vertical drift, use `elecDrift`
```

Comment:
For both HD and VD this should now point to the wirecell sim channels.
The exact naming would need to be checked with an event dump but I believe its `tpcrawdecoder:simpleSC`

Reply:
Hello! 

Yes, that is the correct label. But before the anatree was using the G4 information (by using `largeant` label and I wanted to keep as it was. The G4 label for `sim::SimChannel` appears as `elecDrift`. 

Let me know if I should change it.

Reply:
the default sim channel label with wirecell is `tpcrawdecoder::simpleSC`
Can you check that this sim channel data product is also in the file you're using?

Reply:
Yes, it is also present. 
I've added a new commit and changed the comment for tpcrawdecoder::simpleSC

Reply:
That looks great, thank you!